core/pom.xml

```diff
@@ -2,7 +2,7 @@
   <modelVersion>4.0.0</modelVersion>
   <parent>
     <groupId>io.parsingdata</groupId>
-    <artifactId>metal-parent</artifactId>
+    <artifactId>metal</artifactId>
```

Comment:
I especially want to make this change because when looking at Maven Central or searching for our projects, it is now named _metal-parent_ which is weird. It implies the "parent" part says something about what it does...